summer-of-code/week-02/wk2-homework-submissions/soc_wk2_cert_cristina_tarantino.py

```diff
@@ -491,28 +523,34 @@ def char_frequency_dict_only_present_alphas(text):
 
 # 17. Find out what you can't do with dictionaries. A big one is that they do not have order, so try playing with that.
 def sort_dict_by_numeric_value(dictionary):
+    """Sort a given dictionary by value"""
+
     sorted_dictionary = {}
 
     # https://docs.python.org/3.7/library/stdtypes.html#list.sort
     # sorted(dictionary, key=dictionary.get, reverse=True)
     # returns the array of keys sorted my value. Do to that we use
     # the keyword key (sortBy) that specifies a function of one argument that is used to extract a
     # comparison key from each list element
-    for k, v in [(k, dictionary[k]) for k in sorted(dictionary, key=dictionary.get, reverse=True)]:
-        sorted_dictionary[k] = v
+    for key, val in [(key, dictionary[key]) for key in sorted(dictionary, key=dictionary.get, reverse=True)]:
```

Comment:
Yes i also saw that you have changed the variable indexing from one letters.. Sorry that I missed this one. But yes, using atleast 3to4 char as index names is the best practice preferably, the first 2 or 3 character of what we are referring to,.